tweaks/_main_code.scss

```diff
@@ -384,3 +384,15 @@
         display: none;
     }
 }
+
+@if ($hide-unread-mentions-bar) {
+    div[class*="unreadMentionsBar-"] {
+        display: none;
+    }
+}
+
+@if ($server-list-above and $server-list-above-enable-scrollbar) {
+    .scroller-wrap .scroller {
+        overflow-x: scroll;
```

Comment:
This causes small scrollbars to appear at [the serverlist](https://uwaa.moe/i/SDZZnCC/) and [the chat window](https://uwaa.moe/i/tUSd3UG), which looks stupid. At the very least limit this to only the server list, and hide them if they are not needed.